app/assets/stylesheets/components/_callout-box.scss

```diff
@@ -85,18 +96,31 @@
   }
 
   .info {
-    @include span-columns(10);
-    @include omega();
+    float: left;
+    display: block;
+    margin-right: neat-gutter-width();
+    width: neat-span-width(10);
+    margin-right: 0
+  }
+  
+  .info:last_child {
+    margin-right: 0
   }
 
   .comments {
-    @include span-columns(2);
+    float: left;
+    display: block;
+    margin-right: neat-gutter-width();
```

Comment:
I am going back and forth on this a little bit. I think that if 12 is the default number of columns, we should be specific and use 12, just so it is clear in the future. However, it also looks like for this particular mixin, we are never passing a fewer number of columns.